oxidation/libparsec/crates/crypto/src/rustcrypto/secret.rs

```diff
@@ -101,9 +101,19 @@ impl SecretKey {
     pub fn from_password(password: &str, salt: &[u8]) -> Self {
         let mut key = [0; KEY_SIZE];
 
-        ARGON2
-            .hash_password_into(password.as_bytes(), salt, &mut key)
-            .expect("Invalid salt");
+        if cfg!(feature = "test-unsecure-but-fast-secretkey-from-password") {
```

Comment:
Should add a comment to explain 1) why this is needed 2) what algorithm we use instead

Reply:
The feature name do not suffice ?

If no, would this suffice ?

```suggestion
   // During test we want to skip the `argon2` algorithm for hashing the password
   // Because it take sometime.
   // Force we replace argon with a very basic algorithm that copy the `password + salt` to the key first bytes.
   if cfg!(feature = "test-unsecure-but-fast-secretkey-from-password") {
```

Reply:
The feature name is good, but it is much better with your comment ;-)

(especially given the replacement algorithm is totally insecure and hence it's better for the reader to confirm this is intended)